db/prefix.go

```diff
@@ -25,6 +25,10 @@ const (
 	SYS_CurrentBlock      DataEntryPrefix = 0x40
 	SYS_CurrentBookKeeper DataEntryPrefix = 0x42
 
+	// NAME
+	NS_Registrant DataEntryPrefix = 0x50
```

Comment:
Why need two prefixes? Is it enough if only use an item in database, such as key: Prefix + name, value: programhash?

Reply:
We need to resolve fast not only name -> public key but public key -> name too to check that this public key doesn't have name registered already

Reply:
Can resolve?